### ISSUE TITLE
Feedback - Excellent Sample!

### ISSUE THREAD

**Issue by snerks** [ServiceStackApps/typescript-redux#1]
Great to see an in-depth React/Redux example that uses Typescript and VS2015 - I've been porting ES6 versions so far.

Also - an interesting introduction to JSPM for me.

One minor blip I encountered (in case anyone else sees similar) was that JSPM on the Windows CMD line decided I did not have Git. I messed with the PATH - still no joy.

When I tried the Git Bash shell instead (installed with Git Desktop for Windows), JSPM started working smoothly.

**Comment by mythz** [comment]
Hi thx for the feedback!

I've recently moved from ES5 (i.e. no transpilers) and static bower imports, but since I'm moving to TypeScript made sense to use a module loader and JSPM seems like the optimal choice atm (tho knowing node.js - there can be a better one next week :) - so far haven't had any issues with JSPM except for the multiple requests requiring an interim bundle, but that's due to how React is packaged in npm so can't complain.

I've gone through the tutorial on a couple of different pc's but haven't seen this path error and have GitHub Desktop for Windows installed on both. Did you install Node via the [Node.js Tools for VS.NET?](https://www.visualstudio.com/en-us/features/node-js-vs.aspx) maybe that would resolve the issue?

**Comment by snerks** [comment]
I installed Node "standalone" from the LTS installer at [Node JS website](https://nodejs.org/en/).

I also have the Node version that comes as a side-effect of installing VS2015.

I haven't seen issues with direct use of LTS NPM in the Windows CMD shell.

To be clear - the issue I saw was using non-admin CMD shell on Windows to issue JSPM commands. Git Bash works fine with JSPM. 

Bonus: "Paste" works as desired in Git Bash - which is nice.
